specification/metrics/semantic_conventions/http-metrics.md

```diff
@@ -4,7 +4,7 @@ linkTitle: HTTP
 
 # Semantic Conventions for HTTP Metrics
 
-**Status**: [Experimental][DocumentStatus]
+**Status**: [Experimental, partial feature-freeze][DocumentStatus]
```

Comment:
Should this be "Mixed" then?

Reply:
Nope. I think "Mixed" means the doc has a mixture of [Lifecycle status](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/document-status.md#lifecycle-status) (Experimental/Stable/Deprecated), while we only have "Experimental" here.

[Feature-freeze](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/document-status.md#feature-freeze) is orthogonal to Lifecycle status.